### PR TITLE
DOC-4989-Modified-WinUI-Content-Set1

### PR DESCRIPTION
**Pull request by laksmipriyamurugesan**
documentation(DOC-4989): Reviewed

### CODE REVIEW

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

**Comment by laksmipriyamurugesan** [comment]
rebuild

File: winui/Cartesian-Charts/Axis/Types.md [hunk 1]
```diff
@@ -266,7 +266,7 @@ chart.XAxes.Add(primaryAxis);
 
 ### Interval
 
-In [DateTimeAxis](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeAxis.html), intervals can be customized by using the [Interval](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeAxis.html#Syncfusion_UI_Xaml_Charts_DateTimeAxis_Interval) and [IntervalType](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeAxis.html#Syncfusion_UI_Xaml_Charts_DateTimeAxis_IntervalType) properties. For example, setting [Interval](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeAxis.html#Syncfusion_UI_Xaml_Charts_DateTimeAxis_Interval) as 6 and [IntervalType](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeAxis.html#Syncfusion_UI_Xaml_Charts_DateTimeAxis_IntervalType) as [Months](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.DateTimeIntervalType.html#Syncfusion_UI_Xaml_Charts_DateTimeIntervalType_Months) will consider 6 months as interval.
```
Comment (mohamedks) [inline winui/Cartesian-Charts/Axis/Types.md:269]: Do not change numbers.

**Review by mohamedks — APPROVED** [review]

**Pull request merged**